default.nix

```diff
@@ -438,6 +438,12 @@ let overrideCabal = pkg: f: if pkg == null then null else lib.overrideCabal pkg
     overrideForGhc7 = haskellPackages: haskellPackages.override {
       overrides = self: super: {
         cereal = dontCheck super.cereal; # cereal's test suite requires a newer version of bytestring than this haskell environment provides
+        ghcjs-dom = overrideCabal super.ghcjs-dom (drv: {
```

Comment:
Perhaps add a comment explaining why this complexity exists? That way, in the future, someone can easily know when it can be dropped.